gns3fy/api.py

```diff
@@ -309,7 +309,8 @@ def _verify_before_action(self):
             if _err:
                 raise ValueError(f"{_err}")
 
-            extracted = [node for node in _response.json() if node["name"] == self.name]
+            extracted = [node for node in _response.json()
```

Comment:
The formatting you have set is `black`?. Basically the formatting and linting is:

- Formatting: `black`
- Linting: `flake8`.

Plus the is modified to a better readability, for example in vscode:

```json
    "python.linting.flake8Args": [
        "--max-line-length=88"
    ],
```

Something along those lines are reformatting this lines into new lines, can you take a look on that?

Reply:
No idea here, I'll modify them on my vscode setup

Reply:
cool, so once you have pushed the code with the formatting that it had I will merge it to the `develop` branch